emuu-client/src/components/UserAuthentication/RegisterScreen.js

```diff
@@ -27,45 +31,84 @@ function Register() {
+   });
 
 }
 
+
+
+
   //registration form
   return (
 
 
     <ThemeProvider theme={theme}>
 
-      <div className="col-sm-6 offset-sm-3">
 
-      <h1>Register</h1>
-       <form id="registerForm" method="POST">
 
-       <input type= "text" value = {firstName} onChange ={(e) =>setFirstName(e.target.value)} className= "form-control" placeholder = "First Name" />
+      <h1>Register</h1>
+       <form id="registerForm" method="POST" class="was-validated">
+
+        {/*First Name */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "text" value = {firstName} onChange ={(e) =>setFirstName(e.target.value)} className= "form-control" placeholder = "First Name" required />
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Please fill out this field.</div>
+       </div>
        <br />
-       <input type= "text" value = {lastName} onChange ={(e) =>setLastName(e.target.value)} className= "form-control" placeholder = "Last Name" />
+
+        {/*Last Name */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "text" value = {lastName} onChange ={(e) =>setLastName(e.target.value)} className= "form-control" placeholder = "Last Name" required/>
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Please fill out this field.</div>
+       </div>
        <br />
-       <input type= "text" value = {userName} onChange ={(e) =>setUserName(e.target.value)} className= "form-control" placeholder = "User Name" />
+
+        {/*User Name */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "text" value = {userName} onChange ={(e) =>setUserName(e.target.value)} className= "form-control" placeholder = "User Name" required/>
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Please fill out this field.</div>
+       </div>
        <br />
-       <input type= "text" value = {email} onChange ={(e) =>setEmail(e.target.value)} className= "form-control" placeholder = "Email Address" />
+
+        {/*Email */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "text" value = {email} onChange ={(e) =>setEmail(e.target.value)} className= "form-control" placeholder = "Email Address" required/>
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Please fill out this field.</div>
+       </div>
        <br />
-       <input type= "password" value = {password} onChange ={(e) =>setPassword(e.target.value)} className= "form-control" placeholder = "Password" />
+
+        {/*Password */}
+       <div className="col-sm-6 offset-sm-3">
+       <input type= "password" value = {password} onChange ={(e) =>setPassword(e.target.value)} className= "form-control" placeholder = "Password" required/>
+       <div class="valid-feedback">Valid.</div>
+       <div class="invalid-feedback">Password must be at least 6 characters long!</div>
+       </div>
```

Comment:
I don't understand why you're making your own text inputs. Every component library as built in tools for this